src/main/webapp/i18n/de/assessment.json

```diff
@@ -55,9 +55,9 @@
                     "score": "Punkte",
                     "duration": "Dauer",
                     "assessmentType": "Bewertungsart",
-                    "student": "Student/in",
-                    "assessor": "Prüfer/in",
-                    "assessors": "Prüfer/innen",
+                    "student": "Studierende:r",
+                    "assessor": "Bewertende Person",
+                    "assessors": "Bewertende Personen",
```

Comment:
IMO "Person" as a fallback does not sound too good, such sentences could be a little more context sensitive (e.g. "bewertet von" wherever this might make sense), but this is probably very time consuming to do everywhere so I'm fine with your solution, lmk though if I should try to find a way to adaopt my suggestion (here or in a follow up PR).

Reply:
Thanks for the feedback 😊 
"Bewertet von" is a good idea, I refactored it in d7fba8b.